fusor-ember-cli/app/mirage/factories/deployment.js

```diff
@@ -31,8 +31,8 @@ export default Mirage.Factory.extend({
   custom_preprend_name: null,
   enable_access_insights: true,
   openstack_undercloud_password: "dummy password",
-  upstream_consumer_uuid: "7ffddefd-aacb-4192-a999-01beb7c2e473",
-  upstream_consumer_name: "tsanders-rhci",
+  upstream_consumer_uuid: null,
+  upstream_consumer_name: null,
```

Comment:
@isratrade that is correct. If there is already an upstream_consumer_uuid we assume there are subscriptions in the database and a manifest has already been imported. Therefore, we skip the login screen to disallow the user from uploading a new manifest potentially unentitling a bunch of systems in the process. 

The flow is as follows:

- If there are NO subscriptions i.e. this is your very first deployment, it will behave as it used to. You have 1 of 2 options either login with your portal credentials and go the connected route OR choose to upload a manifest and go the disconnected route.

- If there are already subscriptions, i.e. this is your second or subsequent deployment, you can only use what has already been imported. If you need more you will need to go to the Satellite pages to manage your manifests.